content/docs/csidriver/release/powerflex.md

```diff
@@ -2,20 +2,7 @@
 title: PowerFlex
 description: Release notes for PowerFlex CSI driver
 ---
-
```

Comment:
Are the notes at the bottom of this page still relevant? The image version csi-vxflexos-2.11.0 does not look right.

Reply:
as per PO instructions, we're going to address this separately from the patch changes. It's interesting that there are Notes on the different drivers as well, so we will need to look at these items holistically.